src/test/java/com/openshift/internal/restclient/DefaultClientTest.java

```diff
@@ -165,18 +165,6 @@ public void clientShouldNotEqualClientWithDifferentUrl() throws Exception {
 				.isNotEqualTo(new DefaultClient(new URL("http://localhost:8443"), null));
 	}
 	
-	@Test
-	public void clientShouldNotEqualClientWithDifferentStrategy() throws Exception {
-		DefaultClient tokenClient = new DefaultClient(baseUrl, null);
-		tokenClient.setAuthorizationStrategy(new TokenAuthorizationStrategy("aToken", "aUser"));
-
-		DefaultClient basicAuthClient = new DefaultClient(baseUrl, null);
-		basicAuthClient.setAuthorizationStrategy(new BasicAuthorizationStrategy("aUser", "aPassword", "aPassword"));
-
-		assertThat(tokenClient).isNotEqualTo(basicAuthClient);
-	}
-
-	@Test
 	public void client_should_equal_client_with_same_TokenAuthStrategy_with_different_token() throws Exception {
```

Comment:
@jcantrill nope, that's on purpose. See the method name, it says that 2 clients with different strategies should not be equal, that's exactly the opposite of what I wanted to change, so the test has to disappear.